pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -380,11 +381,12 @@ func (r *Builder) mapFirmware(vm *model.Workload, cluster *model.Cluster, object
 	}
 	switch biosType {
 	case Q35Ovmf:
-		smmEnabled := true
-		features.SMM = &cnv.FeatureState{
-			Enabled: &smmEnabled,
-		}
-		firmware.Bootloader = &cnv.Bootloader{EFI: &cnv.EFI{}}
```

Comment:
I think you'll need to add `fallthrough` here if you want to switch to the next block or do `case Q35Ovmf, Q35SecureBoot:`